cmd/eksctl/get.go

```diff
@@ -3,14 +3,16 @@ package main
 import (
 	"fmt"
 	"os"
+	"strings"
+	"time"
 
+	"github.com/weaveworks/eksctl/pkg/eks"
 	"github.com/weaveworks/eksctl/pkg/printers"
 
-	"github.com/spf13/cobra"
-
+	awseks "github.com/aws/aws-sdk-go/service/eks"
```

Comment:
good idea to call this differently, I just got a little confused reading the other file... would you mind to rename this everywhere else?

Reply:
Maybe in a separate PR :)

Reply:
Done, #142